versioned_docs/version-1.6(WIP)/faq.md

```diff
@@ -16,46 +16,61 @@ In short, a [custodian cell](depAndWthd.md#custodian-cell) is an asset held in t
 
 We're actively exploring ways to improve the withdrawal experience and support fast withdrawal. Please stay tuned.
 
 ----
 
-### Why does it require an amount of 63 CKBs to an intermediate address to move funds back to Layer 1 when using Yokai? 
+### Transferring funds to layer-1 must send tokens to the intermediate address
+
+Q: Why does it require an amount of 63 CKBs to an intermediate address to move funds back to Layer 1 when using Yokai? 
 
-This is a constraint of the current release of Godwoken (L2). Godwoken uses a withdrawal recipient account (actually a cell in Nervos' term) on Nervos CKB (L1) to act as an identity authenticator. Godwoken authenticates the receiver's identity by using an account signature provided by the recipient and then sends corresponding withdrawal funds. The recipient account requires at least 63 CKBs to be stored on Nervos CKB (L1). Those 63 CKBs are still yours and can be claimed back if you don't need this account later. [Actually, anything stored on Nervos CKB (L1) requires an amount of CKBs (1 CKB = 1 Byte) for its storage space. L1 storage is validated by global consensus, thus it's scarce and needs to be used carefully.](https://docs.nervos.org/docs/basics/concepts/economics)
+A: This is a constraint of the current release of Godwoken (L2). Godwoken uses a withdrawal recipient account (actually a cell in Nervos' term) on Nervos CKB (L1) to act as an identity authenticator. Godwoken authenticates the receiver's identity by using an account signature provided by the recipient and then sends corresponding withdrawal funds. The recipient account requires at least 63 CKBs to be stored on Nervos CKB (L1). Those 63 CKBs are still yours and can be claimed back if you don't need this account later. [Actually, anything stored on Nervos CKB (L1) requires an amount of CKBs (1 CKB = 1 Byte) for its storage space. L1 storage is validated by global consensus, thus it's scarce and needs to be used carefully.](https://docs.nervos.org/docs/basics/concepts/economics)
 
 This constraint is a temporary mechanism and will be removed in a future release. Godwoken is the world's first rollup running on a UTXO blockchain. The answer is not ready-made, and we have to explore different ways to find the best design. It's your feedback and patience that help us improve Godwoken and make it better in future iterations. Thank you.
 
 ----
 
-### How to get Layer 1 Block Number?
+### Failed to transfer token via Metamask on Godwoken
```

Comment:
Support for native token transfer
Previously when using Metamask on Godwoken, it was necessary to add a custom ERC20 contract for pCKB to send pCKB tokens. This requirement is no longer required. pCKB is now treated as a native token on Godwoken and works out of the box with Metamask.

- https://github.com/nervosnetwork/godwoken-polyjuice/pull/173
- https://github.com/nervosnetwork/godwoken/pull/788
- https://github.com/nervosnetwork/godwoken-web3/pull/505